src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$card-square: 98px;
```

Comment:
```suggestion
$cell-width: 100px;
```

Reply:
I went with 98px as it has a 1px border.